openoereb/pyramid_oereb_mfp
# Adapt topic page #1215

addresses https://github.com/openoereb/pyramid_oereb/issues/1215
TO TEST:
- use PR https://github.com/openoereb/pyramid_oereb/pull/1328
- turn off pdfA. For instructions see: #77
